packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/readme.md

```diff
@@ -20,7 +20,7 @@ The following folders contain the generated code:
 The Code can be regenerated by running the following command:
 
 ```shell
-yarn schematics @ama-sdk/sdk:typescript-core --spec-path [path to your swagger file]
+yarn schematics @ama-sdk/schematics:typescript-core --spec-path [path to your swagger file]
```

Comment:
Can be fixed by `yarn create @ama-cli typescript --spec-path [path to your swagger file]`